spec/requests/api/v0/articles_spec.rb

```diff
@@ -269,6 +269,15 @@
         expect(response.parsed_body.size).to eq(1)
       end
 
+      it "returns recent articles" do
```

Comment:
Should this spec maybe also generate an article that won't be returned because it's not recent?

Reply:
Well, I believe it would return all articles sorted by publish date up to the pagination limit (30 by default), so unless we modify the API call to return fewer results per page it will still return any article regardless of how old it is. Does that seem like a behavior we should avoid?

Reply:
Ah, maybe I just didn't quite understand what "recent" means in this context, so all good.